cgi/product_multilingual.pl

```diff
@@ -271,6 +271,7 @@ ($product_ref)
 					if $log->is_info();
 				$product_ref = init_product($User_id, $Org_id, $code, $country);
 				$product_ref->{interface_version_created} = $interface_version;
+				$product_ref->{owner} = $Owner_id;
```

Comment:
This should only be set if we are on the producer platform and $Owner_id is defined. When a contributor adds a product on the public platform, we should not set $product_ref->{owner}